main/lsp/requests/code_action.h

```diff
@@ -14,6 +15,12 @@ class CodeActionTask final : public LSPRequestTask {
     std::unique_ptr<ResponseMessage> runRequest(LSPTypecheckerInterface &typechecker) override;
 
     bool canUseStaleData() const override;
+
+private:
+    std::vector<std::unique_ptr<TextDocumentEdit>> getMoveMethodEdits(const LSPConfiguration &config,
+                                                                      const core::GlobalState &gs,
+                                                                      const core::lsp::MethodDefResponse *definition,
```

Comment:
I believe that we know that the pointer is non-null, and I'm a stickler against raw pointers when possible. If this signature works, I'd love to use it:

```suggestion
                                                                      const core::lsp::MethodDefResponse &definition,
```

I hope it's as easy as changing the call sites to `..., *definition, ...`